models/src/main/thrift/content/v1.thrift

```diff
@@ -998,6 +998,10 @@ struct ContentFields {
     43: optional string shortSocialShareText
 
     44: optional string socialShareText
+
+    45: optional string bodyText
+
+    46: optional i32 charCount
```

Comment:
Is it worth calling it `bodyTextCharCount` or similar? There also be charCount for the body field?